mkdocs.yml

```diff
@@ -50,6 +51,7 @@ watch:
 
 nav:
     - Home: index.md
+    - Blocks List: blocks_list.md
```

Comment:
Can we nest this under home so it isn't at the top level? Also, I have a weak opinion that we title the page `Blocks Catalog`, but I leave the decision up to you @ahuang11 
```suggestion
    - Home: index.md
        - Blocks Catalog: blocks_list.md
```

Reply:
Seems a bit strange to me. Maybe I can append it to the README script generator.

![image](https://user-images.githubusercontent.com/15331990/208488016-29431e63-d62b-4ef7-855a-52dc129c4159.png)

Reply:
Okay, I inserted it under getting started:
![image](https://user-images.githubusercontent.com/15331990/208489664-2bd9a101-533c-441c-9d54-2b68866ea798.png)

Reply:
Also, nested the flows into their own section
![image](https://user-images.githubusercontent.com/15331990/208489970-be9be511-7ee4-4928-b084-c4b8a6dcece0.png)